daisy_workflows/image_build/enterprise_linux/enterprise_linux_uefi.wf.json

```diff
@@ -33,6 +33,10 @@
     "rhel_uefi": {
       "Value": "true",
       "Description": "Create a RHEL UEFI image."
+    },
+    "destination_prefix": {
+      "Required": true,
+      "Description": "GCS path to export image metadata to"
```

Comment:
i'm not sure why this is passed in. can't we deduce from the workflow that it should be placed alongside the image? or do we want to publish it somewhere else?

Reply:
We want to place with the same gcs path with image. however the destination is defined in the "export-image" step.
But looks like it's not easy to construct the output path directly. So I thought it's may be easier to pass in the path.

`gs://gce-image-archive/centos-uefi/centos-7-v${build_date}/root.tar.gz
gs://gce-image-archive/centos-mbr/centos-8-mbr-v${build_date}/root.tar.gz`

Reply:
that will require a lot of changes to the infrastructure surrounding these buils i think. let's discuss that offline too and revisit